sdk/messaging/azeventhubs/internal/errors.go

```diff
@@ -176,15 +176,16 @@ func GetRecoveryKind(err error) RecoveryKind {
 	}
 
 	// check the "special" AMQP errors that aren't condition-based.
-	if errors.Is(err, amqp.ErrLinkClosed) || IsQuickRecoveryError(err) {
+	if IsQuickRecoveryError(err) {
```

Comment:
It's nice that this simplified. One less error type to check for the same condition.